src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
Good job! but it's required for this task to use only class components, and you'll have the opportunity to implement the functional components in the future, so please make sure to update this.

Reply:
@thankgodr great job implementing the changes requested however you forgot to convert your app component to be class-based, kindly look into it